lang/cs/build.props

```diff
@@ -76,75 +77,4 @@ under the License.
     <!--The root directory of the REEF source tree. -->
     <REEF_Source_Folder>$([System.IO.Path]::GetFullPath($(SolutionDir)\..\..))</REEF_Source_Folder>
   </PropertyGroup>
-
-  <!--
-       ########################################################################
-         Extract the REEF Version from the POM file 
-       ######################################################################## 
-  -->
-  <!--A Task that extracts the version from the pom in a given folder.-->
-  <UsingTask
-    TaskName="GetMavenProjectVersion"
-    TaskFactory="CodeTaskFactory"
-    AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v4.0.dll" >
-    <ParameterGroup>
-      <SnapshotNumber/>
-      <ProjectFolder Required="true"/>
-      <Version Output="true"/>
-      <NugetVersion Output="true"/>
-    </ParameterGroup>
-    <Task>
-      <Reference Include="System.Xml"/>
-      <Reference Include="System.Xml.Linq"/>
-      <Using Namespace="System"/>
-      <Using Namespace="System.IO"/>
-      <Using Namespace="System.Linq"/>
-      <Using Namespace="System.Xml.Linq"/>
-      <Code Type="Fragment" Language="cs">
-        <![CDATA[
-          Version = XDocument.Load(Path.Combine(ProjectFolder, "pom.xml")).Descendants()
-            .Where(x => x.Name.ToString().Contains("version"))
-            .FirstOrDefault().Value;
-          var snapshotNumberAsString = ($(SnapshotNumber) >= 0 && $(SnapshotNumber) <=9) ? "0" + $(SnapshotNumber) : $(SnapshotNumber).ToString();
-          NugetVersion = $(IsSnapshot) ? Version + "-" + snapshotNumberAsString : Version.Replace("-SNAPSHOT","");
-        ]]>
-      </Code>
-    </Task>
-  </UsingTask>
-  <!--
-    A Target that extract the REEF version from the pom and makes it available as REEF_Version
-  -->
-  <Target Name="ExtractPOMVersion">
```

Comment:
I've just noticed that this target is removed from build.props but it still has `InitialTargets="ExtractPOMVersion"` in its header. Should it be removed?